apps/web/src/lib/components/settings/settings-content.svelte

```diff
@@ -139,41 +169,27 @@
   <SettingsItemGroup title="Hide furigana">
     <ButtonToggleGroup options={optionsForToggle} bind:selectedOptionId={hideFurigana} />
   </SettingsItemGroup>
-  <SettingsItemGroup title="Hide furigana style">
+  <SettingsItemGroup title="Hide furigana style" tooltip={furiganaStyleTooltip}>
     <ButtonToggleGroup options={optionsForFuriganaStyle} bind:selectedOptionId={furiganaStyle} />
   </SettingsItemGroup>
-  <SettingsItemGroup title="Persistent storage">
+  <SettingsItemGroup
+    title="Persistent storage"
+    tooltip="Indicates if unlimited Storage for this Page is enabled."
```

Comment:
Don't think the description is accurate, think it's self-explanatory but if you really wanna include it can refer Yomichan's description on persistent storage

Reply:
its fine i removed it - this option is most likely anyays not applicable under "normal" circumstances